docs/integrations/dbt.md

```diff
@@ -43,27 +43,38 @@ Consider the following when using a dbt project:
 
 ## How to use SQLMesh incremental models with dbt projects
 
-Incremental loading is a powerful technique when datasets are large and recomputing tables is expensive. SQLMesh offers first-class support for incremental models, and its approach differs from dbt's.
-
-SQLMesh automatically detects and offers to backfill missing time intervals for incremental models. dbt's incremental logic does not support intervals and is not compatible with SQLMesh.
+Incremental loading is a powerful technique when datasets are large and recomputing tables is expensive. SQLMesh offers first-class support for incremental models, and its approach differs from dbt's in that SQLMesh will always run incrementally, even the first time, detecting and backfilling each missing time interval.
 
 This section describes how to implement SQLMesh incremental models in a dbt-formatted project.
 
-### dbt's incremental logic
-dbt's incremental logic is implemented with jinja blocks gated by `{% if is_incremental() %}`. 
+### Incremental types
 
-Existing uses of these blocks do not need to be removed from the dbt project's models, but SQLMesh will ignore them.
+SQLMesh supports two approaches to implement [idempotent](../concepts/glossary.md#idempotency) incremental loads: 
 
-### SQLMesh's incremental logic
-SQLMesh's incremental logic is implemented in dbt projects with jinja blocks gated by `{% if sqlmesh is defined %}`.
+* Using merge (with the sqlmesh [`incremental_by_unique_key` model kind](../concepts/models/model_kinds.md#incremental_by_unique_key)) 
+* Using insert-overwrite/delete+insert (with the sqlmesh [`incremental_by_time_range` model kind](../concepts/models/model_kinds.md#incremental_by_time_range))
 
-SQLMesh supports two approaches to implement [idempotent](../concepts/glossary.md#idempotency) incremental loads: 
-- Using merge (with the sqlmesh [`incremental_by_unique_key` model kind](../concepts/models/model_kinds.md#incremental_by_unique_key)) 
-- Using insert-overwrite/delete+insert (with the sqlmesh [`incremental_by_time_range` model kind](../concepts/models/model_kinds.md#incremental_by_time_range))
+#### Incremental by unique key
+
+To enable incremental_by_unique_key incrementality, make sure the model configuration contains the following:
+
+* `unique_key`
+* `materialized` type of `incremental`
+* Either no `incremental_strategy` or `incremental_strategy` of `merge`
+
+#### Incremental by time range
+
+To enable incremental_by_time_range incrementality, make sure the model configuration contains the following:
+
+* `time_column` (see [`time column`](../concepts/models/model_kinds.md#time-column) for details)
+* `materialized` type of `incremental`
+* `incremental_strategy` of either `insert_ovewrite` or `delete+insert`
```

Comment:
Want to make a note here that `insert_overwrite` and `delete+insert` are synonyms and the approach taken is dependent on the engine being targeted?

Reply:
Good idea. Added.